client/hbs/no-admin.hbs

```diff
@@ -1,11 +1,3 @@
-{{> head.hbs}}
-<body>
-	<h1>Please <a href="/google/login">log in</a> to access this page.</h1>
-	<h3>If you encountered this page by accident, you may hit the back arrow. Otherwise, you will be automatically redirected in a few seconds</h3>
-	<script>
-		setTimeout(function() {
-			window.location.href="/google/login"
-		}, 4000)
-	</script>
-</body>
-
+<script>
+window.location = "/google/login";
```

Comment:
I disagree actually. I think there should be a warning in case a non-admin ever encounters that page. Because otherwise they'll just keep trying to log in with no clear reasoning.